marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/impl/sm-es-impl.xqy

```diff
@@ -233,6 +234,7 @@ declare function es-impl:get-definition-properties(
               => map:with("pathExpression", $path-expression)
               => map:with("propertyPath", $property-path)
               => map:with("isIndexed", $is-indexed)
+              => map:with("allowsMultiple", $allows-multiple)
```

Comment:
I would call this "allowsMultipleValues" to make it explicit what "multiple" refers to.

Reply:
renamed